CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue with the owners of this repository before making a change.
```

Comment:
Thanks for adding `CONTRIBUTING.md`!

I would say prior discussion is not necessary for all pull requests.
It would imho make sense if the planned pulled request introduced breaking changes or new features requiring lots of dev work.